src/main/java/net/snowflake/client/jdbc/SnowflakeConnectString.java

```diff
@@ -135,6 +135,12 @@ public static SnowflakeConnectString parse(String url, Properties info) {
         // if it's a global url
         parameters.put("ACCOUNT", account);
       }
+
+      if (Strings.isNullOrEmpty(account)) {
```

Comment:
Please double check this is always true otherwise it may create a regression. I mean do we see account name in the URL in all cases?

Reply:
The JDBC driver requires a connection string with syntax `jdbc:snowflake://<account_identifier>.snowflakecomputing.com/?<connection_params>` 

https://docs.snowflake.com/en/developer-guide/jdbc/jdbc-configure#jdbc-driver-connection-string